ext/libclementine-common/core/logging.cpp

```diff
@@ -21,7 +21,7 @@
 #include <cxxabi.h>
 
 #include <QtGlobal>
-#ifdef Q_OS_UNIX
+#ifdef HAVE_BACKTRACE
 #include <execinfo.h>
```

Comment:
`execinfo.h` also may not be correct. FindBacktrace recommends generating a file that includes `${Backtrace_HEADER}`
https://cmake.org/cmake/help/v3.0/module/FindBacktrace.html

Reply:
Yes, that's why I was asking... I went for the trivial change...

Reply:
Solved. I think.